### PR TITLE
adding Semantic Arts, the gist ontology, and books on ontology engineering & DCA

### PR DESCRIPTION
**Pull request by pwin**
DCA = data-centric architecture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ Looking for something but can't find it? Add it to the ["Does it exist"](https:/
 ## Companies
 Companies or businesses selling products with a primary focus on semantic web technology
 
+- [Semantic Arts](https://semanticarts.com) - Enterprise information systems based on flexible data structures and deep semantics.
```
Comment (maximveksler) [inline README.md:202]: This one belongs to industry, please add it to the bottom of the list.

File: README.md [hunk 2]
```diff
@@ -678,6 +681,8 @@ OS - OpenSource
 
 ## Books
 
+- [The Data-Centric Revolution](https://www.amazon.co.uk/Data-Centric-Revolution-Restoring-Enterprise-Information/dp/1634625404)
+- [Ontology Engineering](https://www.morganclaypool.com/doi/abs/10.2200/S00834ED1V01Y201802WBE018)
```
Comment (maximveksler) [inline README.md:685]: Thank you for the contribution, please use append() to existing list.

File: README.md [hunk 3]
```diff
@@ -594,6 +596,7 @@ OS - OpenSource
 
 ## Ontologies
 
+- [gist](https://www.semanticarts.com/gist/) - minimalist enterprise upper ontology - max coverage, fewest primitives, least ambiguity.
```
Comment (maximveksler) [inline README.md:599]: Thank you for the contribution, please use append() to existing list.

File: README.md [hunk 4]
```diff
@@ -464,6 +465,7 @@ ARCHIVE - inactive projects or old academic projects that may lack soruce code
 
 ### Blogs
 
+- [Semantic Arts Data-Centric Architecture blog](https://www.semanticarts.com/blog/)
```
Comment (maximveksler) [inline README.md:468]: Thank you for the contribution, please use append() to existing list.

**Comment by pwin** [comment]
HI @maximvelsler

[Semantic Arts](https://semanticarts.com) - Enterprise information
systems based on flexible data structures and deep semantics. is
actually a business that sells products with a primary focus on
semantic web technology.

I've made the other changes as suggested


Thanks

@pwin

**Comment by maximveksler** [comment]
@pwin it might be that the applied changes were not pushed? I don't I seem them in the PR.

Under companies we prefer to have business that contribute to the development of semantic web technologies, I kindly request that your listing be appended to the enterprise section.

**Comment by pwin** [comment]
Hi Maxim
Semantic arts do develop semantic technologies. The company develops and
maintains an upper ontology for enterprise data, and it also is developing
products for constructing and maintaining a model driven data centric
architecture
That's why I was suggesting that it should be listed in that place
Cheers
Peter

**Comment by zacharywhitley** [comment]
It's not a perfect distinction between companies and industry. The idea was to identify that some companies were using semantic web technologies but where their main purpose was something else like manufacturing cars, banking, etc. There's obviously some overlap. If you are representing your company I would defer to listing it where you'd like but ask that it only be listed in  one or the other and not both.

**Comment by pwin** [comment]
afaik it is only in the Companies section

**Pull request merged**